fdmi/plugins/m0scheduler

```diff
@@ -17,128 +17,277 @@
 # For any questions about this software or licensing,
```

Comment:
Similar to rename comment below: https://github.com/Seagate/cortx-motr/pull/846/files#r670193482

Can we make this generic python sample to show how to start m0fdmiapp_kv and extract key, value and have a placeholder comment like `# your code goes here for processing FDMI records.`
This way end users will be able to extend this for multisite example in our case or any other fdmi use case. This script, for other developers can give a view of how to extract motr configs and how to start fdmi app.

one alternate name can be m0pyfdmiapp_kv ?

Reply:
It is difficult to make an elegant and meaningful name here. I propose to use something simple, like sample-plugin or something with its given name after a bird (like the colibri) or fish or octopus ;)

Reply:
Plugin sounds confusing from an motr client api users perspective to me, calling it fdmi app that can receive fdmi events sounds more intuitive. Also now I realise should we move it to `fdmi/st` where there is similar other sample?

Reply:
Agreed with the fdmi app. ST means system tests, they are testing how the bigger parts of a system interact with each other. I propose to create a new directory, something like the example, tutorials, etc.